apps/chameleon/components/ui/carousel.tsx

```diff
@@ -32,6 +33,18 @@ type CarouselContextProps = {
 
 const CarouselContext = React.createContext<CarouselContextProps | null>(null)
 
+const carouselVariant = cva({
```

Comment:
this is not how we should handle extending the carousel.  You need to take the carousel atoms and make it into your own molecule that has the proper extensions.  See all the other files outside of the UI file.